libsolidity/ast/Types.cpp

```diff
@@ -362,6 +368,69 @@ MemberList::MemberMap Type::boundFunctions(Type const& _type, ASTNode const& _sc
 	if (auto refType = dynamic_cast<ReferenceType const*>(&_type))
 		typeLocation = refType->location();
 
+	return usingForDirectives | ranges::views::filter([&](UsingForDirective const* _directive) -> bool {
+		// Convert both types to pointers for comparison to see if the `using for`
+		// directive applies.
+		// Further down, we check more detailed for each function if `_type` is
+		// convertible to the function parameter type.
```

Comment:
This needs to be updated now that "further down" is not a part of the function:
```suggestion
		// Convert both types to pointers for comparison to see if the `using for` directive applies.
		// Note that at this point we don't yet know if the functions are actually usable with the type.
		// `_type` may not be convertible to the function parameter type.
```

Reply:
updated